pkgs/development/python-modules/togglCli/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, buildPythonPackage, fetchPypi, pythonPackages }:
+
+
+buildPythonPackage rec {
+  pname = "togglCli";
+  version = "2.1.0";
+
```

```suggestion

  disabled = pythonOlder "3.5";
```

See https://github.com/AuHau/toggl-cli/blob/master/setup.cfg#L10.